duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/inputscreen/ui/view/InputModeWidget.kt

```diff
@@ -285,6 +303,10 @@ class InputModeWidget @JvmOverloads constructor(
         }
     }
 
+    fun setScrollPosition(position: Int, positionOffset: Float) {
```

Comment:
Is it really worth creating a function if we only use it once and contains a single line of code?